engines/wintermute/utils/utils.cpp

```diff
@@ -44,7 +44,7 @@ void BaseUtils::swap(int *a, int *b) {
 
 //////////////////////////////////////////////////////////////////////////
 float BaseUtils::normalizeAngle(float angle) {
-	while (angle > 360) {
+	while (angle > 359) {
```

Comment:
That would work if the angle was a `int`, but since this is a `float` this is incorrect. For example now we would get -0.2 instead of 359.8.

Reply:
The following loop covers it:
https://github.com/scummvm/scummvm/blob/8469b8749c9e710992d22bdfa07e6afcff9cfd4d/engines/wintermute/utils/utils.cpp#L50-L52

Reply:
Right. That will teach me to look beyond the displayed diff! :-P
I was expecting to see `>= 360` and didn’t look further.